test/Github.Api.Contract.test.js

```diff
@@ -0,0 +1,25 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const chai = require('chai');
+const { listPublicEventsSchema } = require('./schema/ListPublicEvents.schema');
+
+const { expect } = chai;
+chai.use(require('chai-json-schema'));
+
+const urlBase = 'https://api.github.com';
+
+describe('Given event Github API resources', () => {
+  describe('When wanna verify the List public events', () => {
```

Comment:
when who? you should use `I want to...`

Reply:
@sorregom D: así estaba, eso se copiaba directamente del taller, ya lo corrijo